microsoft-edge/devtools-guide-chromium/index.md

```diff
@@ -3,7 +3,7 @@ description: Get to know the Microsoft Edge (Chromium) Developer Tools
 title: Microsoft Edge (Chromium) Developer Tools overview  
```

Comment:
Change # h1 top-of-page title from:
Microsoft Edge (Chromium) Developer Tools overview
to:
Microsoft Edge Developer Tools overview

AND in first sentence of body, change:
When you install Microsoft Edge
to:
When you install Microsoft Edge (Chromium)